### PR TITLE
update ice cream extractor naming, update rest_extractor example dataset creation

### PR DESCRIPTION
**Pull request by maur1**

### CODE REVIEW

**Review by chrfla-cognite — APPROVED** [review]

**Comment by maur1** [comment]
Could you approve @einarmo ?

**Comment by einarmo** [comment]
@mathialo?

File: ice-cream-factory-datapoints-extractor/pyproject.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "ice cream factory datapoints extractor"
+name = "rest_extractor"
```
Comment (mathialo) [inline ice-cream-factory-datapoints-extractor/pyproject.toml]: why rename to rest_extractor? I would find that a bit confusing especially as it is not using the rest extension.

I think `ice_cream_extractor` or something similar is a better name, and entrypoint 🙂

Comment (maur1) [reply ice-cream-factory-datapoints-extractor/pyproject.toml]: done

**Review by mathialo — APPROVED** [review]

**Pull request merged**